sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar-project.properties
+sonar.organization=fedora-infra
+sonar.projectKey=fedora-infra/noggin
+sonar.sources=.
+sonar.host.url=https://sonarcloud.io
+sonar.login=a36fe78f64abe3b8e6db0df06a6b399c2abc2e24
+sonar.test.exclusions=**/noggin/tests/*
```

Comment:
I'm not sure why the double star at the beginning. The tests are under `noggin/tests` so this should probably be `noggin/tests/**`, no?
Also, are you sure it's `sonar.test.exclusions` and not `sonar.exclusions`? I haven't found a clear distinction between the two in [the doc](https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/).

Reply:
There is a difference between "test" exclusions and "regular" exclusions, I ran this file a few different ways to determine how to write the exclusions properly.